packages/create-block/README.md

```diff
@@ -99,6 +99,53 @@ $ npm run packages-update
 ```
 Updates WordPress packages to the latest version. [Learn more](/packages/scripts#packages-update).
 
+## External Templates
+Since version 0.12.0 it is possible to use external templates hosted on NPM. These templates need to contain `.mustache` files that will be used in the scaffolding and one `template.json` for the metadata. 
+
+### Availabe Variables: 
+- `namespace`
+- `slug`
+- `title`
+- `textdomain`
+- `description`
+- `category`
+- `dashicon`
+- `license`
+- `licenseURI`
+- `namespaceSnakeCase`
+- `slugSnakeCase`
+
+### `template.json`
+```json
+{
+    "defaultValues": {
+        "namespace": "create-block",
+        "slug": "esnext-example",
+        "title": "ESNext Example",
+        "description":
+            "Example block written with ESNext standard and JSX support – build step required.",
+        "dashicon": "smiley",
+        "category": "widgets",
+        "author": "The WordPress Contributors",
+        "license": "GPL-2.0-or-later",
+        "licenseURI": "https://www.gnu.org/licenses/gpl-2.0.html",
+        "version": "0.1.0"
+    },
+    "outputFiles": [
+        ".editorconfig",
+        ".gitignore",
+        "editor.css",
+        "src/edit.js",
+        "src/index.js",
+        "src/save.js",
+        "$slug.php",
+        "style.css",
+        "readme.txt"
+    ],
+    "wpScriptsEnabled": true
```

Comment:
Thinking about it now when documented, it should be opt-out rather than opt-in, we want people to use `wp-scripts` :)